templates/waggle-dance-federation-remote.yml.tmpl

```diff
@@ -0,0 +1,4 @@
+  - name: ${prefix}
+    access-control-type: READ_ONLY
+    database-prefix: ${prefix}_
+    remote-meta-store-uris: thrift://${metastore_host}:${metastore_port}
```

Comment:
Would need to support `metastore_tunnel` as well

Reply:
idea here is we can build federation yaml from parts, if we need ssh tunnels for remote regions, we can create separate issue and add corresponding template and logic.